src/lms-harmonizer/tests_integration_pgsql/pgsql_orchestrator.py

```diff
@@ -44,27 +42,31 @@ def _run(config: PgsqlServerConfig, command: List[str]):
 
 
 def run_harmonizer(config: PgsqlServerConfig):
-    args_in = [
-        "--server",
-        config.server,
-        "--port",
-        config.port,
-        "--dbname",
-        config.db_name,
-        "--username",
-        config.username,
-        "--password",
-        config.password,
-        "--engine",
-        "postgresql",
-    ]
-    args_parsed = parse_main_arguments(args_in)
-    run(args_parsed)
```

Comment:
This was causing most tests to fail due to connections remaining open.

Reply:
How odd that it was working fine on my system but failing on yours. Though I do wonder if this might relate in some way to the odd behavior I was seeing with database rollbacks.